src/api/app/views/layouts/webui/webui.html.haml

```diff
@@ -1,5 +1,5 @@
 !!!
-%html{ lang: 'en' }
+%html{ lang: 'en', "data-bs-theme-from-user": User.session&.color_theme || "system" }
```

Comment:
I am thinking that we should actually keep the beta flag for this specifically, only display this if a person has explicitly enabled a beta of the feature. The reason being is that I think there are going to be some issues still and I don't really want to risk interrupting the work of people who aren't willing to try things out. If we only have this data attribute for beta users, and only display the dropdown in user settings for beta users then we won't have this issue.

Reply:
OMG, I just removed all that work hahaha.
Ok, it makes sense to me... that's why I started with that in the first place.
I'll add that after all the rest is settled, WDYT?

Reply:
Yeah, I want it to one day move it out of beta, but I think that it's not now and we are better safe than to disturb the software so many people use for work :P